test/gtest/core/test_allreduce.cc

```diff
@@ -91,6 +91,8 @@ class test_allreduce : public UccCollArgs, public testing::Test {
                 UCC_CHECK(ucc_mc_alloc((void**)&dsts[r],
                                        count * sizeof(typename T::type),
                                        UCC_MEMORY_TYPE_HOST));
+                dsts[r] = (typename T::type *) ucc_malloc(count * sizeof(typename T::type), "dsts[%d] buf", r);
```

Comment:
there can only be 2 args to ucc_malloc, 2nd arg should be const char *, ie string